src/awkward/contents/unionarray.py

```diff
@@ -189,7 +190,7 @@ def simplified(
         index = ak.index.Index64.empty(length, backend.index_nplike)
         contents = []
 
-        for i, self_cont in enumerate(contents):
+        for i, self_cont in enumerate(self_contents):
```

Comment:
This had been an error in the translation of `simplify_uniontype` to `simplified`. I don't see any similar errors.